pulsar-client-cpp/include/pulsar/ProducerConfiguration.h

```diff
@@ -501,6 +506,12 @@ class PULSAR_PUBLIC ProducerConfiguration {
      */
     bool isChunkingEnabled() const;
 
+    ProducerConfiguration& setAccessMode(const ProducerAccessMode& mode);
+
+    ProducerAccessMode getAccessMode() const;
+
+    bool isLazyStartPartitionedProducers() const;
```

Comment:
Unfortunately we already have `getLazyStartPartitionedProducers`, though the name might not be good. Please don't introduce a new getter.